### ISSUE TITLE
Add Eclipse

### ISSUE THREAD

**Issue by JustRed23** [kennytv/list-of-shame#81]
Theres no reason to ever use Eclipse. Intellij > Eclipse

**Comment by Andre601** [comment]
I beg to differ.
Eclipse has its fair share of uses and some of the more popular and/or well made plugins out there are made with it.

To name one: BkCommonLib

**Comment by trainb0y1** [comment]
Yeah, although I firmly belive IntelliJ > Eclipse, I don't think it belongs on a list of shame.

~~not that my opinion matters lol~~

**Comment by KeinOptifine** [comment]
To add my unasked for opinion to the mix: Don't judge people for the tools their using. If they can work with em its fine

**Comment by Fox2Code** [comment]
My opinion is that Intellij IDEA need to be added too, they made a perfect IDE, then JetBrain wanted to improve it, they made it worse instead by introducing 2000 warnings that are not even problems and that make suggestion that make the code slower.

Intellij IDEA 2016 was a perfect experience, but Intellij IDEA 2022 is something else.
I need to disable a lot of warnings because they reduce my code quality.
Some warnings are great at increasing performances, but a lot decrease them, especially the one added after 2020.

**Comment by GNosii** [comment]
IntelliJ IDEA is straight up an lag machine for my computer rn.

**Comment by andriy332** [comment]
> IntelliJ IDEA is straight up an lag machine for my computer rn.

agree

**Comment by LaserSlime** [comment]
> IntelliJ IDEA is straight up an lag machine for my computer rn.

Glad to know i'm not the only one. This is one of the reasons why I switched back to eclipse both of the times where I tried to switch to IntelliJ after some nerds on some discord server told me to.

**Comment by yum13241** [comment]
`neovim`: Am I a fucking joke to you?
